Dockerfile

```diff
@@ -12,7 +12,7 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-jre-alpine
+FROM tomcat:9-jre8-alpine
```

Comment:
I thought there were some concerns regarding the docker container size. Is this not an issue anymore?

Reply:
Ok, that makes sense to me. I thought you had changed this before from the tomcat to the alpine image, that's why I was confused.